Assets/Settings/Localization/Strings/Strings_es.asset

```diff
@@ -609,7 +609,7 @@ MonoBehaviour:
     m_Metadata:
       m_Items: []
   - m_Id: 7870824155160576
-    m_Localized: Ajustes de cambio y trazo
+    m_Localized: 
```

Comment:
Is this deliberately removed? (there are a few more like this; I'm not highlighting all of them